src/setup/rgw_service

```diff
@@ -39,7 +39,7 @@ done
 
 function start_rgw {
   machine_id=$(cat /etc/machine-id)
-  exec sudo /usr/bin/radosgw -f --name client.rgw-"$index" -c "$rgw_config_file" --no-mon-config &> "$log_file" &
+  exec /usr/bin/radosgw -f --name client.rgw-"$index" -c "$rgw_config_file" --no-mon-config &> "$log_file" &
```

Comment:
@rahul-tripathi-git  do we need '&' at the end of this exec command ?

Reply:
This file will be removed with PR https://github.com/Seagate/cortx-rgw-integration/pull/38.
& will send the execution to background. But since we have sleep infinity at K8s level, service process going to background is fine.